OData/test/E2ETest/WebStack.QA.Test.OData/OpenType/TypedTest.cs

```diff
@@ -399,6 +399,146 @@ public static void UpdateConfiguration(HttpConfiguration configuration)
             }
         }
 
+        [Fact]
+        public async Task PatchOpenComplexTypeProperty()
+        {
+            foreach (string routing in Routings)
+            {
+                await ResetDatasource();
+
```

Comment:
It's ok. I test for three scenarios.
Basically, it's not necessary for e2e test cases to add // Arrange // Act // Assert.